examples/export-data/models/custom-event-type.ts

```diff
@@ -0,0 +1,29 @@
+import mongoose, { Schema } from "mongoose";
+import { CustomEventType } from "../utils/types";
+import ProjectModel from "./project";
+
+export const CustomEventTypeSchema = new mongoose.Schema<CustomEventType>({
```

Comment:
remember the custom event type id comment I left in main.ts - when you are copying over the custom event types, you will want to explicitly set the _id field so add _id to this schema (else you won't be able to map the event type ids to their categories and subcategories)